group-generators/helpers/data-providers/eas/index.ts

```diff
@@ -0,0 +1,123 @@
+  ): Promise<FetchedData> {
+    const dataProfiles: FetchedData = {};
+
+    for await (const item of this._getAttestations(params)) {
+      dataProfiles[item.recipient] = 1;
+    }
+
+    return dataProfiles;
+  }
+
+  public async getAttestationRecipientCount(
+    params: GetAttestationParams
+  ): Promise<number> {
+    const attestations = await this.getAttestationRecipients(params);
+    return Object.keys(attestations).length;
+  }
+
+  public async getAttestationValues(
+    params: GetAttestationParams
+  ): Promise<FetchedData> {
+    const dataProfiles: FetchedData = {};
+
+    for await (const item of this._getAttestations(params)) {
+      const value = isNaN(Number(item.value)) ? 1 : Number(item.value);
+      dataProfiles[item.recipient] = value;
+    }
+
+    return dataProfiles;
+  }
+
+  public async getAttestationValuesCount(
+    params: GetAttestationParams
+  ): Promise<number> {
+    const attestations = await this.getAttestationValues(params);
+    return Object.keys(attestations).length;
+  }
+
+  private async *_getAttestations(params: QueryParams) {
+    const baseUri = `https://${
+      params.network ? `${params.network}.` : ""
+    }easscan.org/graphql`;
+    const client = new GraphQLClient(baseUri);
+    let offset = 0;
+    let hasNext = true;
+
+    while (hasNext) {
+      const res = await client.request(`
+        {
+          attestations(take: ${defaultLimit} skip: ${offset} where: {
+            revoked: {
+              equals: false
+            }
+            attester: {
+              equals: "${params.attester}",
+              mode: insensitive
+            }
+            schemaId: {
+              equals: "${params.schema}",
+              mode: insensitive
+            }
+            OR: [
+              {
+                expirationTime: {
+                  gt: ${Math.floor(Date.now() / 1000)}
+                }   
+              }
+              {
+                expirationTime: {
+                  equals: 0
+                }   
+              }
+            ]
+            }) {
+            id
+            attester
+            recipient
+            revoked
+            data
+            decodedDataJson
+            time
+            timeCreated
+            schema {
+              id
+              schema
+            }
+          }
+        }
+      `);
+
+      if (!params.key && !params.value) yield* res.attestations;
+
+      yield* res.attestations.filter((i: any) => {
+        const values = JSON.parse(i.decodedDataJson);
+        const val = values.find((v: any) => {
+          if (params.key && !params.value)
+            return v.name.toLowerCase() === params.key.toLowerCase();
+
+          if (!params.key && params.value)
+            return v.value.value.toLowerCase() === params.value.toLowerCase();
+
+          return (
+            v.name.toLowerCase() === params.key?.toLowerCase() &&
+            v.value.value.toLowerCase() === params.value?.toLowerCase()
```

Comment:
I have an error here using the example you give in the interfaceSchema:
```
{
      schema: "0x85500e806cf1e74844d51a20a6d893fe1ed6f6b0738b50e43d774827d08eca61",
      attester: "0x0fb166cDdF1387C5b63fFa25721299fD7b068f3f",
      key: "Gm",
      value: "True",
}
```
Error: "v.value.value.toLowerCase is not a function"

It's because the v.value.value is of type boolean, that's why the toLowerCase doesn't work.
To fix this put a .toString() before each time you use toLowerCase() function

Otherwise the examples you put in the PR description work well 👍